rendercv/data_models.py

```diff
@@ -1589,65 +1583,65 @@ def get_a_sample_data_model(
         "projects": [
             NormalEntry(
                 name="Multi-User Drawing Tool",
-                date="2004",
+                date="2004",  # I would omit start/end dates and replace with a project link as the date is largely irrelevant
+                # project_link = "github.com/username/repo"
                 highlights=[
                     (
                         "Developed an electronic classroom where multiple users can"
                         ' view and simultaneously draw on a "chalkboard" with each'
-                        " person's edits synchronized."
+                        " person's edits synchronized"
                     ),
-                    "Used C++ and MFC.",
+                    "Tools Used: C++, MFC",
                 ],
             ),
             NormalEntry(
                 name="Synchronized Calendar",
                 start_date="2003",
                 end_date="2004",
+                # project_link = "github.com/username/repo",
                 highlights=[
                     (
                         "Developed a desktop calendar with globally shared and"
                         " synchronized calendars, allowing users to schedule meetings"
-                        " with other users."
+                        " with other users"
                     ),
-                    "Used C#.NET, SQL, and XML.",
+                    "Tools Used: C#, .NET, SQL, XML",
                 ],
             ),
             NormalEntry(
                 name="Operating System",
-                date="2002",
+                date="2002",  # would replace project dates with a project link as the date is largely irrelevant
+                # project_link = "github.com/username/repo",
```

Comment:
We shouldn't include this line.

Reply:
Let's keep the date to show users that only years are acceptable too. I agree, it seems irrelevant.